data/simple-icons.json

```
@@ -18231,6 +18231,11 @@
 		"hex": "990000",
 		"source": "https://www.weasyl.com"
 	},
+	{
+		"title": "Web Awesome",
+		"hex": "F36944",
+		"source": "https://fontawesome.com/icons/web-awesome?f=brands&s=solid"
```

```suggestion
		"source": "https://fontawesome.com/icons/web-awesome"
```

Remove unnecessary search parameters.